doc: wifi: Update radio test documents to reflect new commands (#14410)
nrfconnect/sdk-nrf: doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

Wi-Fi
-----

* Added:

  * :kconfig:option:`NRF_WIFI_FW_PATCH_DFU` Kconfig option that enables DFU support for nRF70 Series devices.
    This allows firmware patches for signed images to be sent to multi-image targets over Wi-Fi.

>   * Support for CSP package.

Reviewer: ```suggestion
  * Support for the CSP package.
```

>   * Application of edge backoff is modified. It used to be subtracted from regulatory power limit. Now, backoff is subtracted from data rate dependent power ceiling.

Reviewer: ```suggestion
  * Modification to the application of edge backoff.
    Previously, it was subtracted from the regulatory power limit.
    Now, backoff is subtracted from the data rate-dependent power ceiling.
```

    * Added ``promiscuous_set`` extension to the Wi-Fi command line.
      It adds the subcommand to configure Promiscuous mode.

* :ref:`wifi_radio_test` sample:

  * Updated:

>     * Added support for run time configuration of antenna gain and edge backoff parameters.

Reviewer: ```suggestion
    * Added support for runtime configuration of antenna gain and edge backoff parameters.
```
